src/main/java/ch/example/threetypesofexception/adapters/rest/exceptionhandling/ErrorResponseDto.java

```diff
@@ -1,4 +1,4 @@
-package ch.example.threetypesofexception.interfaces.restexceptionhandling;
+package ch.example.threetypesofexception.adapters.rest.exceptionhandling;
```

Comment:
In this case I would prefer my solution. I don't like the name "adapters". We could merge the outermost ring into "infrastructure" or some other word, but adapters is not a good name for this ring, from my point of view.